docs/API.md

```diff
@@ -0,0 +1,353 @@
+
+### unless
+
+#### `unless : ((a -> Boolean) | Pred) -> (a -> b) -> a -> a | b`
+There may come a time when you need to adjust a value when a condition is false, that is where `unless` can come into play. Just provide a predicate function (a function that returns a Boolean) and a function to apply your desired modification. This will get you back a function that when you pass it a value, it will evaluate it and if false, will run your value through the provided function. Either the original or modified value will be returned depending on the result of the predicate. Check out `when` for a negated version of this function.
+
+### when
+
+#### `when : ((a -> Boolean) | Pred) -> (a -> b) -> a -> b | a`
+There may come a time when you need to adjust a value when a condition is true, that is where `when` can come into play. Just provide a predicate function (a function that returns a Boolean) and a function to apply your desired modification. This will get you back a function that when you pass it a value, it will evaluate it and if true, will run your value through the provided function. Either the original or modified value will be returned depending on the result of the predicate. Check out `unless` for a negated version of this function.
+
+## Predicates
+
+All functions in this group have a signature of `a -> Boolean` and are easily used with the many predicate based functions that ship with `crocks`, like `safe`, `ifElse` and `filter` to name a few. They also fit naturally with the `Pred` ADT. Below is a list of all the current predicates that are included with a description of their truth:
+
+* `isApplicative`: an ADT that provides `map`, `ap` and `of` functions
+* `isApply`: an ADT that provides `map` and `ap` functions
+* `isArray`: Array
+* `isBoolean`: Boolean
+* `isEmpty`: Empty Object, Array or String
+* `isFunction`: Function
+* `isFunctor`: an ADT that provides a `map` function
+* `isInteger`: Integer
+* `isMonad`: an ADT that provides `map`, `ap`, `chain` and `of` functions
+* `isMonoid`: an ADT that provides `concat` and `empty` functions
+* `isNil`: undefined or null
+* `isNumber`: Number that is not a NaN value, Infinity included
+* `isObject`: Plain Old Javascript Object (POJO)
+* `isSemigroup`: an ADT that provides a `concat` function
+* `isSetoid`: an ADT that provides an `equals` function
+* `isString`: String
+* `isTraversable`: an ADT that provides `map` and `traverse` functions
+
+## Point-free
+
+While it can seem natural to work with all these containers in a fluent fashion, it can get cumbersome and hard to get a lot of reuse out of. A way to really get the most out of reusability in Javascript is to take what is called a point-free approach. Below is a small code same to contrast the difference between the two calling styles:
```

Comment:
I was not sure whether to add them here or keep them in the README, so they were purposely inconsistent for option; does your new issue #42 makes these suggestions obsolete, or what do you think?